template_feed/Microsoft.DotNet.Test.ProjectTemplates/content/MSTest-CSharp/.template.config/template.json

```diff
@@ -28,9 +44,18 @@
         {
           "choice": "netcoreapp1.1",
           "description": "Target netcoreapp1.1"
+        },
+        {
+          "choice": "netcoreapp2.0",
+          "description": "Target netcoreapp2.0"
         }
       ],
+      "replaces": "netcoreapp1.0",
       "defaultValue": "netcoreapp1.0"
+    },
+    "IsNetStandard2": {
+      "type": "computed",
+      "value": "(Framework == 'netcoreapp2.0' || Framework == 'netstandard2.0')"
     }
```

Comment:
Nit: 'netstandard.20' is not a valid choice.
Guessing either this, or the previous, similar nit has an incorrect choice.